content/defects-tech-debt.rst

```diff
@@ -218,6 +218,7 @@ Things that can be done in this day:
 * push or send upstream your local changes to open source projects
 * fix a bug in an upstream open source project.
 
+After each day, please send email feedback to the team to talk about what you have done.
```

Comment:
maybe ... after each "hacking session"

that is if you are doing some nice things outside of the happy hacking session, you are encourage to let the team know

I know that on the ticket, there were other notes about updating this section... like : is ok to work in another day, not on the first Wednesday